### ISSUE TITLE
Feature: More control over news feeds 

### ISSUE THREAD

**Issue by cleargif** [cryptoinsightio/cryptoinsight-client#12]
As a user, I want more control over the content in the news feed.

- Basic filters for coins, hashtags, publishers etc
- Pull in custom feeds from 3rd parties
- Turn feeds on or off

**Comment by cryptoinsightio** [comment]
Yep, this is planned. We're going to allow users to add in their own feeds (right now any kind of RSS feed is supported), we'll see if we can throw a filter in there too.
